code/modules/events/meteors.dm

```diff
@@ -163,7 +162,8 @@ var/global/list/thing_storm_types = list(
 	meteor_wave(rand(10, 20), types = thing_storm_types[storm_name], offset_origin = 150, offset_dest = 230) //Much more clement
 
 /datum/event/thing_storm/end()
-	command_alert("The station has cleared the [storm_name].", "Meteor Alert")
+	spawn(450)
```

Comment:
`45 SECONDS`

also if you're just going to do this to all thing_storms, why not bake in the delay at a higher level and just have the message vary?

Reply:
Don't know what a thing_storm is so I'd rather not touch it if that's the case

Reply:
Oh I see what you mean now., I'll fix it

Reply:
it's not so simple